src/app/styles/tailwind/ui/colors.js

```diff
@@ -0,0 +1,58 @@
+const colors = {
+  primary: {
+    100: '#f2f9ff',
+    200: '#dff0ff',
+    300: '#b3daff',
+    400: '#4faaff',
+    500: '#006ed6',
+    600: '#00529e',
+    700: '#003a71',
+    800: '#002547',
+    900: '#03172a',
```

Comment:
HEX values could be fine for now, but if you want to use colors with opacity, like text-primary-100/50(opacity: 50%) then it will not work. Consider using something like
```
function generatePallete(name, { exclude = [] } = {}) {
  const colors = [50, 100, 200, 300, 400, 500, 600, 700, 800, 900, 950].filter((n) => !exclude.includes(n));

  return Object.fromEntries(
    colors.map((n) => [
      n,
      ({ opacityValue, opacityVariable }) => {
        const variable = `--color-${name}-${n}`;
        if (opacityValue !== undefined && !isNaN(opacityValue)) {
          return `rgba(var(${variable}), ${opacityValue})`;
        }
        return `rgba(var(${variable}), var(${opacityVariable || '--tw-opacity'}, 1))`;
      },
    ]),
  );
}
```
where --color-${name}-${n} is CSS variable that looks like `--color-primary-100: 255, 255, 255;`
You may skip this for now, but remember that you can't use color opacity